examples/cockroachdb/README.md

```diff
@@ -0,0 +1,19 @@
+# CockroachDB on Kubernetes as a PetSet
+
+This example deploys CockroachDB on Kubernetes as a PetSet. Note that
+CockroachDB already does replication and thus should not be deployed on
+a persistent volume which already replicates internally. Best performance
+would dictate use of local storage, which is currently not an option (at
+least not as a PetSet).
+
+## Running on minikube
+
+Follow the steps in [minikube.sh](minikube.sh) (or simply run that file).
+
+## Simulating failures
+
+When all (or enough) nodes are up, simulate a failure like this:
+
+```shell
```

Comment:
Done. I didn't go all out because `bash` isn't the right language for these tests (and we already have them elsewhere).
Also not too much about tradeoffs etc - I think for the toy example it's fine to assume that people generally have an idea of what they're toying with.